### PR TITLE
reaggregate, launch rerun of models

### PR DESCRIPTION
**Pull request by aappling-usgs**
models are running now. did some checks - the aggregated data look like i expected

retro:
image

analysis:
image

med:
image

a long:
image

### CODE REVIEW

File: 2_munge/src/aggregate_inputs.R [hunk 1]
```diff
@@ -54,124 +54,85 @@ aggregate_nwm <- function(ind_file, raw_ind_file, remake_file, sites_file, gd_co
...
   }
 
+  # aggregate according to the groups set above
+  agg_nwm <- grouped_nwm %>%
+    summarise(
+      n = length(flow),
+      flow = if(n==24/valid_time_step) mean(flow) else NA) %>%
+    ungroup() %>%
+    filter(!is.na(flow))
+
```
Comment (jzwart) [inline 2_munge/src/aggregate_inputs.R]: very nice!

File: 3_forecast/src/prep_inputs.R [hunk 2]
```diff
@@ -60,9 +62,11 @@ prep_inputs <- function(
     EGRET::populateDaily(qConvert=q_divisor, verbose = FALSE)
   #Daily <- readNWISDaily("06934500","00060","1979-10-01","2010-09-30")
   if(any(is.na(flow$Q)) || any(is.na(flow$LogQ))) {
+    noQ_dates <- filter(flow_conc, is.na(Q) | is.na(LogQ)) %>% pull(Date)
```
Comment (jzwart) [inline 3_forecast/src/prep_inputs.R]: could you specify `dplyr::filter`

File: 3_forecast/src/prep_inputs.R [hunk 3]
```diff
@@ -106,7 +110,7 @@ prep_inputs <- function(
   # make sure the merge will go smoothly
   flow_conc <- left_join(conc, flow, by='Date')
   if(any(is.na(flow_conc$Q))) {
-    noQ_dates <- filter(flow_conc, is.na(Q))$Date
+    noQ_dates <- filter(flow_conc, is.na(Q)) %>% pull(Date)
```
Comment (jzwart) [inline 3_forecast/src/prep_inputs.R]: again `dplyr::filter`

File: 2_munge/src/aggregate_inputs.R (hunk 1 above)
Comment (jzwart) [reply 2_munge/src/aggregate_inputs.R]: also some `dplyr::filter` in this script

**Comment by aappling-usgs** [comment]
Oops, yes, forgot the dplyr::filter qualifier. Will add.

**Pull request merged**